cloud/deploys/dev_azure/README.md

```diff
@@ -24,15 +58,8 @@ docker push <DOCKER_USERNAME>/<DOCKER_REPO_NAME>:<IMAGE_TAG>
 You can do this one of two ways. Terraform deploy or update via the azure 
 portal 
 
-### Update via the .tfvars File and deploy
-After that you can change your .tfvars file to point to the docker tag you set
-
-```
-docker_repository_name = <DOCKER_REPO_NAME>
-docker_username        = <DOCKER_USERNAME>
-```
-
-deploy via `terraform apply`
+### Update via the deploy script
+TODO: Document this?
```

Comment:
TODO

Reply:
hahahahaha this was for future me i didn't test the local deploy in this

Reply:
leaving this out and someone can add the documentation for it when they write it!

Reply:
we should probably write a deploy script that builds the docker image and then calls deploy